var/spack/repos/builtin/packages/synapse-tool/package.py

```diff
@@ -0,0 +1,75 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+import os
+
+from spack import *
+
+
+class SynapseTool(CMakePackage):
+    """Synapse format utilities.
+    """
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/hpc/synapse-tool"
+    url      = "ssh://bbpcode.epfl.ch/hpc/synapse-tool"
+
+    version('dev-201806',
+            '',
+            commit='8c4bbe548006d7221f215f32c077338f169ba015',
```

Comment:
Any way to have better version name? Also, I believe the empty `''` not required (?).

Reply:
I was seeing if I can force it to get a sha for the commit/version with the empty string?

I'm not really a fan of a "rolling" target based on the latest head, I'll see if we have some "proper" versions, this is similar to how we do it in nix.